recipes/models.py

```diff
@@ -44,7 +44,7 @@ def upvotes(self):
         Return upvote count
         """
         votes = self.votes.through.objects.filter(recipe=self)
-        return votes.filter(up=True).count() - votes.filter(up=False).count()
+        return votes.filter(up=True).count()
```

Comment:
so this would actually return the _total score_ in upvote form, that's why it was the positive votes minus the negative ones. That would give the total. The properly itself may be named inaccurately, but we need this logic somewhere to display the total points based on up and down votes.

Reply:
Sorry I guess I don't understand how upvotes work.  My assumption was that up and down votes were separate.  Would you like me to change it back and modify the test?

Reply:
Thinking about the data model, if upvote and downvote were separate models, we'd need double the database queries to get the total vote count. Voting is essentially a nullable boolean, a vote can have three states; voted up, voted down, or not voted.

So by making the `vote` a single field, with an up, "not up", and null, we cover all the choices but that's all in a single model and thus only a single query.

Reply:
I just didn't know what upvotes are and assumed it was just the positive or 'up' votes.  I can change it back and fix the tests if you want.

Reply:
I changed this back to original